include/lsst/daf/base/DateTime.h

```diff
@@ -57,19 +59,27 @@ namespace base {
 
 class DateTime {
 public:
-    enum Timescale { TAI, UTC, TT };
-    enum DateSystem { JD, MJD, EPOCH };
+    enum DateSystem { JD=0, MJD, EPOCH };
+    enum Timescale { TAI=5, UTC, TT };  // use values that do not overlap DateSystem
+                                        // to avoid confusing one for the other in Python
```

Comment:
Yes, I get that.  But there are still two enums with confusingly similar names.   I'd have preferred a type failure, but Russell tells me that the code will catch my stupidity, so it's good enough.